person_capture/gui_app.py

```
@@ -7979,7 +7999,39 @@ def _repair_wic_saturated_rgb_speckles(self, path: str) -> int:
+            try:
+                hsv = cv2.cvtColor(bgr, cv2.COLOR_BGR2HSV)
+                sat = hsv[:, :, 1].astype(np.int16, copy=False)
+                val = hsv[:, :, 2].astype(np.int16, copy=False)
+                local_sat = cv2.medianBlur(hsv[:, :, 1], 5).astype(np.int16, copy=False)
+                channel_jump = np.maximum(np.maximum(np.abs(b_jump), np.abs(g_jump)), np.abs(r_jump))
+                min_ch = np.minimum(np.minimum(r, g), b)
+                hsv_tiny_seed = (
+                    (local_luma <= 136)
+                    & (sat >= 72)
+                    & (val >= 42)
+                    & ((max_ch - min_ch) >= 34)
```

Ruff (0.15.12)

[error] Undefined name `max_ch`

(F821)